code/__HELPERS/roundend.dm

```diff
@@ -284,6 +284,10 @@
 	standard_reboot()
 
 /datum/controller/subsystem/ticker/proc/standard_reboot()
+#ifdef UNIT_TESTS
+	UNTIL(ready_for_reboot)
+#endif
```

Comment:
Remove this, you probably only needed it for TEST_FOCUS and this gets fixed by https://github.com/tgstation/tgstation/pull/74604

Reply:
```suggestion
```